apps/web/src/components/auth/SignUpForm.tsx

```diff
@@ -19,6 +19,11 @@ type Props = {
 
 export function SignUpForm({ token, email }: Props) {
   const navigate = useNavigate();
+  const [params] = useSearchParams();
+  const code = params.get('code');
+  const next = params.get('next');
+  const isFromVercel = code && next;
```

Comment:
This check should be part of `apps/web/src/api/hooks/use-vercel-integration.ts` instead because it is used multiple times across the codebase (DRY-Principle)